server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -937,4 +937,17 @@ INDEX author_ft USING FULLTEXT (author) WITH (analyzer = 'standard')
                 """.stripIndent()
         ));
     }
+
+    @Test
```

Comment:
I used it to debug, I think analyzer test is enough and this can be removed?

Reply:
Yes lets remove them.